tests/pyccel/test_pyccel.py

```diff
@@ -27,17 +28,36 @@ def get_abs_path(relative_path):
     return os.path.join(base_dir, relative_path)
 
 #------------------------------------------------------------------------------
-def get_exe(filename):
+def get_exe(filename, language=None, prog=False):
     exefile = os.path.splitext(filename)[0]
-    if sys.platform == "win32":
-        exefile = exefile + ".exe"
-    if not os.path.isfile(exefile):
-        dirname = os.path.dirname(filename)
-        basename = "prog_"+os.path.basename(filename)
-        exefile = os.path.join(dirname, os.path.splitext(basename)[0])
+
+    if language == 'python':
+        result = ''
+        if sys.platform == "win32":
+            result = exefile.split('\\')
+            result.insert(-1, "py")
+            if prog is True:
+                result[-1] = "prog_"+result[-1]
+            exefile = "\\".join(result)
+        else:
+            result = exefile.split('/')
+            result.insert(-1, "py")
+            if prog is True:
+                result[-1] = "prog_"+result[-1]
+            exefile = "/".join(result)
+        exefile += '.py'
+        exefile = os.path.normpath(exefile)
```

Comment:
This if/else is not necessary if you use functions from the `os.path` module as is done in the non-python case